package.json

```diff
@@ -59,7 +60,8 @@
   },
   "repository": "guardian/cdk-cli",
   "scripts": {
-    "prepack": "rm -rf lib && tsc -b && oclif-dev readme",
+    "prepack": "rm -rf lib && tsc -b && yarn copy-files && oclif-dev readme",
+    "copy-files": "copyfiles -a -u 1 \"./src/template/**/*\" lib",
```

Comment:
Is `cp` not sufficient to perform this action or is `copyfiles` doing something a bit smarter?

Reply:
When I did some googling about how to solve this, a couple of places suggested to use `copyfiles` over `cp` as it's designed to be OS agnostic.

Reply:
Ah, yeah, that makes sense. In that case go forth!